src/core/templatetags/ascii_encode.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+@register.filter
+def ascii_encode(string):
+    """ ASCII-encode a string, ignoring non-ASCII chars. """
```

Comment:
Don't you think converting those unicode characters to html entities might be a better option?

Reply:
This isn't to convert unicode characters to ASCII in the HTML template (that's displaying fine). It's to look up the corresponding value in a dict of field names that have been ASCII-encoded to avoid saving form field names with non-ascii values. Might be easier to explain in person tomorrow. This bug report describes part of the problem I'm having: https://code.djangoproject.com/ticket/12645#no1. Putting all the changes in place is a big job but I'm getting there.

Reply:
You are using this in the template, not the form. If you use this in the template, like you are, no unicode character will be shown, which may not be what is wanted. I know what to do to encode the unicode to html entities

```string..encode('ascii', 'xmlcharrefreplace')```

Will do the trick